### PR TITLE
Change rate counter

### PR DESCRIPTION
**Pull request by ferri082**

### CODE REVIEW

File: server/src/main/java/umm3601/digitalDisplayGarden/PlantController.java [hunk 1]
```diff
@@ -187,10 +188,9 @@ else if(rating.get("like").equals(false))
             }
         }
 
+        interactions = likes + dislikes + comments;
 
-        out.put("commentCount", comments);
-        out.put("likeCount", likes);
-        out.put("dislikeCount", dislikes);
+        out.put("interactionCount", interactions);
```
Comment (lverns) [inline server/src/main/java/umm3601/digitalDisplayGarden/PlantController.java:191]: We should update the comment on this method to reflect the change in the output.

**Comment by lverns** [comment]
Looks good to me.

**Comment by 15sora2** [comment]
Looks great! Good job!

**Comment by Hoff0899** [comment]
I agree, good work!

**Pull request merged**